saleor/product/migrations/0072_auto_20180929_0707.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.8 on 2018-09-29 12:07
```

Comment:
Could you please rebase with the latest master? I think we are already on 72nd migration.